python/example_code/route53-recovery-cluster/routing_control_states.py

```diff
@@ -114,8 +114,11 @@ def toggle_routing_control_state(routing_control_arn, cluster_endpoints):
     update_state = 'Off' if state == 'On' else 'On'
     print(f"Setting control state to '{update_state}'.")
     response = update_routing_control_state(routing_control_arn, cluster_endpoints, update_state)
-    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
```

Comment:
We can just update it to be as follows:
`if response: print("Success") else: print("Error")`

Reply:
Works for me. Updated.